packages/manager/modules/manager-pci-common/src/api/data/instance.ts

```diff
@@ -35,6 +35,9 @@ export const getInstance = async (
   return data;
 };
 
+/**
+ * If possible use getInstancesByRegion instead
+ */
```

Comment:
Why not deprecating this call and replacing the clients impacted?

Reply:
I am not sure if all non-regionalized routes should be deprecated, if so yes I should mark it.

Replacing all the clients impacted would take a lot of time since in the usages they do not all uses the filter on region.

Reply:
I can also remove the region parameter if that makes sense but would lead to more development if region is not required